lib/packages/core/src/api/__tests__/accountApi.spec.ts

```diff
@@ -4,6 +4,8 @@ import {BurstService} from '../../burstService';
 import {getAccountTransactions} from '../account/getAccountTransactions';
 import {getUnconfirmedAccountTransactions} from '../account/getUnconfirmedAccountTransactions';
 import {getAccountBalance} from '../account/getAccountBalance';
+import { generateSendTransactionQRCodeAddress } from '../account/generateSendTransactionQRCodeAddress';
+import { generateSendTransactionQRCode } from '../account/generateSendTransactionQRCode';
```

Comment:
I think GetSendTransactionQRCode fits better, as it really fetches the code from server

Reply:
I thought we were trying to name the API after the method names in BRS? The url generation is actually the more useful of the two methods, and the one we will use in phoenix (for an IMG tag, you just need the URL, not the actual image).

Reply:
ah, yes-- BRS names, and the other reason is ok too, but it's not really an API function. More kind of util